src/routes/SheltersPege/SheltersPege.js

```diff
@@ -77,34 +100,210 @@ export default class SheltersPege extends Component {
+      phoneNumber,
+      website
+    } = this.state;
+
+    let data = {
+      "shelterID": parseInt(shelterID),
+      "shelterName": shelterName,
+      "address": address + ", " + city + ", " + state + " " + zip,
+      "emailAddress": email,
+      "phoneNumber": parseInt(phoneNumber),
+      "website": website ? website : null,
+    }
+
+    SheltersService.updateShelter(data)
+      .then(res => {
+        this.props.onSubmitSuccess();
+      })
+      .catch(res => {
+        this.setState({ error: res.error });
+      });
+  }
+
+
+  renderShelterName(shelter) {
+    return (
+      <div className='col-md m-1'>
+        <div className='d-flex justify-content-between align-items-center flex-wrap'>
+          <h2>{shelter[0].ShelterName}</h2>
+          {isShelterAdmin(shelter[0].ShelterID) &&
+            <div>
+              <span role='button' className='btn p-2 text-primary' onClick={this.onClickEdit}>
+                <FontAwesomeIcon icon='edit' />
+              </span>
+              <span role='button' className='btn p-2 text-primary' onClick={this.onClickDelete} >
+                <FontAwesomeIcon icon='trash-alt' />
+              </span>
+            </div>
+          }
+        </div>
+        <hr />
+      </div>
+    )
+  }
+
   renderShelter(shelter) {
     log.debug("Calling renderShelter...")
     return (
       <div>
-        <h2>{shelter[0].ShelterName}</h2>
+        <table className='shelterTable'>
+          <tbody>
+            <tr>
+              <th>Address</th>
+              <td>{shelter[0].Address}</td>
+            </tr>
+            <tr>
+              <th>Email</th>
+              <td><a href={"mailto:" + shelter[0].EmailAddress}>{shelter[0].EmailAddress}</a></td>
+            </tr>
+            <tr>
+              <th>Phone number</th>
+              <td><a href={"tel:" + shelter[0].PhoneNumber}>{shelter[0].PhoneNumber}</a></td>
+            </tr>
+            <tr>
+              <th>Website</th>
+              <td><a href={shelter[0].Website}>{shelter[0].Website}</a></td>
+            </tr>
+          </tbody>
+        </table>
         <hr />
-        <div>
-          <table className='shelterTable'>
-            <tbody>
-              <tr>
-                <th>Address</th>
-                <td>{shelter[0].Address}</td>
-              </tr>
-              <tr>
-                <th>Email</th>
-                <td><a href={"mailto:" + shelter[0].EmailAddress}>{shelter[0].EmailAddress}</a></td>
-              </tr>
-              <tr>
-                <th>Phone number</th>
-                <td><a href={"tel:" + shelter[0].PhoneNumber}>{shelter[0].PhoneNumber}</a></td>
-              </tr>
-              <tr>
-                <th>Website</th>
-                <td><a href={shelter[0].Website}>{shelter[0].Website}</a></td>
-              </tr>
-            </tbody>
-          </table>
-        </div>
+      </div>
+    );
+  }
+
+  renderUSStates() {
+    let stateOptions = [<options></options>];
```

Comment:
`<options></options>` can be removed. I added that in there before the states were moved to a JSON, where there is an empty string built in. Also the `<options>` tag doesn't exist, which is totally my bad lol

Reply:
fixed